meta-resin-common/recipes-support/resin-mounts/resin-mounts.bb

```diff
@@ -60,6 +63,8 @@ do_install () {
     install -d ${D}${sysconfdir}/systemd/system/resin-bind.target.wants
     install -c -m 0644 ${WORKDIR}/resin-bind.target ${D}${systemd_unitdir}/system/
 
+    # Create dropin dir for timesyncd.conf mount
```

Comment:
As I see you already have a before attribute in the `resin-ntp-config.service` which makes this drop in useless now.